shared/src/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -611,19 +612,37 @@ public void run() {
         c4ReplListener.statusChanged(c4repl, c4ReplStatus, this);
     }
 
-    private void documentEnded(boolean pushing, String docID, String revID, int flags, C4Error error, boolean trans) {
-        DocumentReplication update = new DocumentReplication((Replicator) this,
-                flags == C4Constants.C4RevisionFlags.kRevDeleted,
-                 pushing, docID, revID, flags, error, trans);
-        if (!pushing && error.getDomain() == LiteCoreDomain && error.getCode() == kC4ErrorConflict) {
-            // Conflict pulling a document -- the revision was added but app needs to resolve it:
-            Log.i(TAG, "%s: pulled conflicting version of '%s'", this, docID);
+    private void documentEnded(boolean pushing, C4DocumentEnded[] documents) {
```

Comment:
Can you sync the logic/implementation with .NET? Basically if the conflict gets resolved, we should clear the error.

Reference: https://github.com/couchbase/couchbase-lite-net/commit/0f929300396b74e5086a31cfa854291af294a8f4